cli/generators/crud/templates/appService.js

```diff
@@ -0,0 +1,81 @@
+import fetch from 'isomorphic-fetch';
+import * as endpoints from './apiEndpoints';
+
+class <%= ucName %>Service {
+  static load<%= pluralizedUcName %>() {
+    const request = new Request(`${endpoints.BASE_URL}${endpoints.GET_<%= pluralizedName.toUpperCase() %>}`, {
+      method: 'GET'
+    });
+
+    return fetch(request).then(response => {
+      return response.json();
+    }).catch(error => {
+      return error;
+    });
```

Comment:
Is is possible to improve these promises?

- Maybe the `.json` portion can be simplified out of the lambda (something like `fetch(request).then(r => r.json())`
- I think that the catch is necessary to avoid rejecting the promise down the line... but isn't that something that we actually want? (So, could we remove the `catch` portion altogether?)

This applies to every example later, I will not comment on all of them so that I don't create a lot of spam.